std/uuid.d

```diff
@@ -732,15 +738,95 @@ public struct UUID
         /**
          * ditto
          */
-        @safe pure nothrow @nogc size_t toHash() const
+       @safe pure nothrow @nogc UUID opAssign(in UUID s)
         {
-            size_t seed = 0;
-            foreach(entry; this.data)
-               seed ^= cast(size_t)entry + 0x9e3779b9 + (seed << 6) + (seed >> 2);
+            ulongs[0] = s.ulongs[0];
+            ulongs[1] = s.ulongs[1];
+            return this;
+        }
 
-            return seed;
+        /**
+         * ditto
+         */
+        @safe pure nothrow @nogc UUID opAssign(ref in UUID s)
+        {
+            ulongs[0] = s.ulongs[0];
+            ulongs[1] = s.ulongs[1];
+            return this;
         }
 
+        /**
+         * ditto
+         */
+        //MurmurHash2
```

Comment:
Compatible with Phobos license? Another Q - doesn't run-time have good hashing maybe fix it there first?